mediathread/settings_shared.py

```diff
@@ -318,6 +318,8 @@ def default_url_processor(url, label=None, request=None):
     'embed_tool_id': 'mediathread',
 }
 
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTOCOL', 'https')
```

Comment:
In the django documentation for this setting, they set the first tuple element to `HTTP_X_FORWARDED_PROTO` instead of `HTTP_X_FORWARDED_PROTOCOL`. I just wanted to make sure that's not a typo.

Searching around I found some random info about an `X-Forwarded-Proto` header in nginx.. is there some place we can see this header when making an HTTP request to mediathread?